Parsers/ASimRegistryEvent/test/VMwareCarbonBlackCloud_vimRegistryEvent_SchemaTest.csv

```diff
@@ -0,0 +1,29 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [RegistryPreviousKey]"
+"(1) Warning: Missing recommended field [RegistryPreviousValueData]"
+"(1) Warning: Missing recommended field [RegistryPreviousValueType]"
+"(1) Warning: Missing recommended field [RegistryPreviousValue]"
+"(1) Warning: Missing recommended field [RegistryValueData]"
+"(1) Warning: Missing recommended field [RegistryValueType]"
+"(1) Warning: Missing recommended field [RegistryValue]"
+"(2) Info: Missing optional field [ActingProcessGuid]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [DstDescription]"
+"(2) Info: Missing optional field [DvcDescription]"
+"(2) Info: Missing optional field [DvcMacAddr]"
+"(2) Info: Missing optional field [DvcOsVersion]"
+"(2) Info: Missing optional field [DvcScopeId]"
```

Comment:
what is this Org_Id? Can we map this to some existing field like ActorScopeId?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/bc585214-a49b-4bda-b1ee-ddc68cdf297b)
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/10a6da94-638c-4ba9-afe7-0c23a46675c3)

Reply:
Done

Reply:
@niralishah-crest  @jayeshprajapaticrest 
In other parser, I suggested to map with DvcScopeId. Can we change this to DvcScopeId for consistency?